iulianIachimov/src/main/java/datacollectiontask/Models/Order.java

```diff
@@ -0,0 +1,53 @@
+package datacollectiontask.Models;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.util.ArrayList;
+import java.util.List;
+
+public class Order {
+    public long id;
+    public LocalDateTime date;
+    public User user;
+    public List<Item> items;
+
+    private static long idCounter;
+
+    public Order(User user, List<Item> items) {
+        this.id = ++idCounter;
+        this.date = LocalDateTime.now();
+        this.user = user;
+        this.items = items;
+    }
+
+    DateTimeFormatter dtf = DateTimeFormatter.ofPattern("hh:mm, MMMM dd, YYYY");
+
+    @Override
+    public String toString() {
+        return "Order{" +
+                "id=" + id +
+                ", date=" + date +
+                ", user=" + user +
+                ", items=" + items +
+                '}';
+    }
+
+    // creating function to filter the orders by type CLOTHES
+    public static List<Order> filterOrderByItemType(List<Order> orderList) {
+        List<Order> orderResult = new ArrayList<>(); // creating a list to store the result
+        for (Order order : orderList) {
+            for (Item item : order.items) {
+                if (Type.CLOTHES == item.type) {
+                    orderResult.add(order);
+                    System.out.println("Order " + order.id + " from " + order.date + " contains type Clothes");
+                    break;
+                }
+            }
+        }
+        return orderResult;
+    }
+}
+
```

Comment:
format

Reply:
Would you please specify what needs to be formatted the code itself or how the result is printed?

Reply:
format code

Reply:
This is done